packages/botonic-plugin-ai-agents/src/index.ts

```diff
@@ -17,7 +26,31 @@ export default class BotonicPluginAiAgents implements Plugin {
   async getInference(
     request: BotContext,
     aiAgentArgs: AiAgentArgs
-  ): Promise<AiAgentResponse> {
-    return this.aiAgentClient.getInference(request, aiAgentArgs)
+  ): Promise<AgenticOutputMessage | undefined> {
+    try {
+      const authToken = isProd ? request.session._access_token : this.authToken
+      if (!authToken) {
+        throw new Error('Auth token is required')
+      }
+
+      const hubtypeClient = new HubtypeApiClient(authToken)
+      const messages = isProd
```

Comment:
We should only initialize the HubtypeApiClient only if its prod, otherwise it's always initialized even if it's not used.